tools/clang/include/clang/SPIRV/SpirvFunction.h

```diff
@@ -99,9 +104,8 @@ class SpirvFunction {
   bool containsAlias; ///< Whether function return type is aliased
   bool rvalue;        ///< Whether the return value is an rvalue
 
-  spv::FunctionControlMask functionControl; ///< SPIR-V function control
```

Comment:
I've removed `functionControl` mask from the class because it was never used (we use control mask of **None**)